packages/terra-site/package.json

```diff
@@ -33,7 +33,7 @@
     "terra-clinical-item-view": "^0.x",
     "terra-content": "^0.x",
     "terra-detail-view": "^0.x",
-    "terra-grid": "^1.0.0",
+    "terra-grid": "^2.0.0",
```

Comment:
The terra-detail-view still has a DevDependency on terra-grid, this was used for examples before they moved to terra-site, but I think there are still tests utilizing the grid. Should terra-detail-view also be bumped, or is it something we can just update in the future?